src/Python/omop_table_examples.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import requests
+import json
+from IPython.display import display
+
+from src.Python.metadata_examples import examples
+
+
+def omop_domain(field):
+    concept_id = int(examples[examples['field_id'] == field]['target_concept_id'].unique())
+    response = requests.get(f'https://athena.ohdsi.org/api/v1/concepts/{concept_id}')
+    concept = json.loads(response.text)
+    domain = concept['domainId']
+    return domain
+
+
+def example_omop_table(field, instance, *value):
```

Comment:
Currently, the function can only process one value. Maybe later we can allow multiple values to be given. So replace `*value` with just `value` for now.